lib/creek/sheet.rb

```diff
@@ -78,12 +80,13 @@ def rows_generator include_meta_data=false
                 y << (include_meta_data ? row : processed_cells)
               elsif (node.name.eql? 'c') and (node.node_type.eql? opener)
                   shared = node.attribute('t').eql? 's'
-                  cell = node.attribute('r')
+                cell_type      = node.attribute('t')
```

Comment:
libxml version on my machine gives wrong values for non-existent attributes (It gives "http://schemas.openxmlformats.org/spreadsheetml/2006/main").
I had to change `node.attribute('t')` to `node.attributes['t']` so cell_type would be nil if 't' attribute doesn't exists.
This fixes my problem of dates not being parsed correctly.